### ISSUE TITLE
Compile libMDDSpaceMouse.a with -fPIC

### ISSUE THREAD

**Issue by bernhard-thiele** [modelica-3rdparty/Modelica_DeviceDrivers#388]
Problem: Linking `Resources/Library/linux64/libMDDSpaceMouse.a/libMDDSpaceMouse.a` fails when trying to make a shared object. 

Solution:  Push a binary which was compiled with the `-fPIC` option.

**Comment by tbeu** [comment]
OT: Prefered workflow on GitHub (instead of creating an self-assigned issue and closing via magic keyword in commit message during direct push to master) could be to create a PR, enable auto-merge with squash and let the CI checks succeed. What do you think? The benefit I see is the green CI.

**Comment by bernhard-thiele** [comment]
You are right. I bypassed that in a hurry, but the workflow with CI is much better.
